src/events/lavalink/rawLavalinkDestroy.ts

```diff
@@ -0,0 +1,19 @@
+import { Events } from '@lib/types/Enums';
+import { LavalinkEvent } from '@utils/LavalinkUtils';
+import { Event, EventStore } from 'klasa';
+
+export default class extends Event {
+
+	public constructor(store: EventStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			emitter: store.client.lavalink!,
+			event: 'destroy'
+		});
+	}
```

Comment:
Use `@ApplyOptions` please

Reply:
Not possible as it's accessing to the `store` variable.

Reply:
I mean it is, `"lavalink"`

Reply:
Not possible as `client.lavalink` is nullable.

Reply:
why tf is client.lavalink nullable